src/olympia/pages/templates/pages/shield_study_14.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Containers</title>
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/shield_study_12/main.css" type="text/css" />
```

Comment:
Made this change ... though I wasn't able to spot-check locally that the pages are still styled correctly. 😒